ngraph/core/src/op/experimental_detectron_generate_proposals.cpp

```diff
@@ -0,0 +1,69 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
```

Comment:
Change copyright year to 2021. The same comment for other files.

Reply:
Fixed.